pkg/pprof/pprof.go

```diff
@@ -1331,6 +1331,11 @@ func sanitizeProfile(p *profilev1.Profile) {
 		if len(x.Value) != vs {
 			return true
 		}
+		for i := range x.Value {
+			if x.Value[i] < 0 {
+				x.Value[i] = 0
+			}
+		}
```

Comment:
This is a little bit more complicated: we already do this sanitisation outside this function, in https://github.com/grafana/pyroscope/blob/main/pkg/pprof/pprof.go#L390-L394

In ad-hoc, profiles travel a different path, and this actually handled here:
https://github.com/grafana/pyroscope/blob/main/pkg/og/structs/flamebearer/convert/convert.go#L205